{"id": "1366401997285576705", "creation": 1614610597.0, "user_id": "884022229", "social_network": "twitter", "nsfw": false, "request": ["b26b9dbc-1c5a-4cd3-a3a6-c2f5a51ad0fc"], "metrics": {"b26b9dbc-1c5a-4cd3-a3a6-c2f5a51ad0fc": {"retweet_count": 16, "reply_count": 1, "like_count": 14, "quote_count": 0}}, "text": "We can\u2019t blame AI when humans mess up https://t.co/rNFD8c0hxh \n\n#MachineLearning #DataScience #Python #AI #100DaysOfCode #DEVCommunity #IoT #flutter #Serverless #CyberSecurity #DeepLearning #COVID19 #vaccine #data #5G #innovation #startups #Entrepreneur #business #Health #tech", "first_save": 1634559108.199414, "hashtags": ["#MACHINELEARNING", "#DATASCIENCE", "#PYTHON", "#AI", "#100DAYSOFCODE", "#DEVCOMMUNITY", "#IOT", "#FLUTTER", "#SERVERLESS", "#CYBERSECURITY", "#DEEPLEARNING", "#COVID19", "#VACCINE", "#DATA", "#5G", "#INNOVATION", "#STARTUPS", "#ENTREPRENEUR", "#BUSINESS", "#HEALTH", "#TECH"]}